bundles/CoreBundle/src/Migrations/Version20221003115124.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20221003115124 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Rename o_id columns to id.';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql("ALTER TABLE pimcore.objects CHANGE o_id id int(11) unsigned auto_increment NOT NULL;");
```

Comment:
Should we be adding the table name like `pimcore.object`? Isn't the database name configurable?

Reply:
DB-name can be just removed as not needed.

Reply:
I´ll remove the database name